common/common.sh

```diff
@@ -417,8 +417,8 @@ bin_output_dmesg() {
   last_dmesg_timestamp
   test_print_trc "$bin_name $parm"
   [[ -e "$bin_name" ]] || block_test "No $bin_name in lkvs, do you compile it?"
-  BIN_OUTPUT=$($bin_name $parm 2>/dev/null)
-  BIN_RET=$?
+  BIN_OUTPUT=$($bin_name "$parm" 2>/dev/null)
```

Comment:
Modification of this function doesn't appear to be directly related to fixing the timestamp bug. Better to put them in another commit or PR.

Reply:
Make sense, I will put them in another commit.

Thanks!